app/assets/stylesheets/gradesheet.css.scss

```diff
@@ -89,7 +93,7 @@ td.focus {
 
 #grades td.edit {
   text-align: center;
-  background-color: white;
+  background-color: #f2f2f2;
```

Comment:
<img width="173" alt="Screenshot 2022-12-31 at 00 24 24" src="https://user-images.githubusercontent.com/9074856/210130408-aa6bfd6e-b021-4fae-b306-8d017495bee0.png">

Nit: the new background-color makes it hard to read text on hover. We should make hover text darker.

Reply:
Turned hover color to the same as background color of submission buttons.

Reply:
I guess that works